modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>typo in two places:</p><ul><li>denote -&gt; denotes</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>